docs/player-states.md

```diff
@@ -0,0 +1,64 @@
+
+# Monitoring Player States  
+
+The player supports lifecycle states (or statuses), which are based on HTML5 video events (e.g., waiting, playing, etc.). Whenever a state change occurs, the player will fire a related event in response.
+
+## Table of Contents
+  - [Registering to State Changes](#registering-to-state-changed)
+  - [Available State Classes](#state-class)
+  - [Available State Types](#state-types)
+  - [Transitioning Between States](#transitions-between-states)
+
+## Registering to State Changes  
+
+When a player changes its state, it fires a `PlayerStateChanged` event. The event data contains the player's old state as well as the new state. By registering to state changes and listening events, you can manage dependent logic accordingly (this applies mostly to UI changes).
+
+The state's you register to listen to include the state class, the state type, and transitions between states. These next section provide additional details.
+
+## Available State Classes  
+
+The player supports a number of state classes. You'll find a detailed list of `State` class APIs [here]().
```

Comment:
@dan-ziv in classes you mean CSS classes?

Reply:
No, my originally intent was the state class.